packages/client-core/src/components/InstanceChat/index.tsx

```diff
@@ -117,8 +118,14 @@ const InstanceChat = (props: Props): any => {
   }, [chatState])
 
   React.useEffect(() => {
-    if (sortedMessages && sortedMessages[sortedMessages.length - 1]?.senderId !== user?.id.value) {
+    if (
+      sortedMessages &&
+      sortedMessages[sortedMessages.length - 1]?.senderId !== user?.id.value &&
+      chatState.messageCreated.value
+    ) {
       setNoUnReadMessage(false)
+      const audio = new Audio(notificationAlertURL)
```

Comment:
we should see if we need to manually clean this up or if it will be handled with the garbage collector, or better yet if we can re-use the audio node